csuntechlab/iSTART
# STR-133 -- FE: Implement Single Question Multiple Choice Slides #6, #7, #15, #27 into Alcohol Module

# Description
Updated quiz template to support all quiz types; Implemented single question quiz slides.
  
# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. Visit the alcohol module.
3. Ensure slides (***6, 7, 15, 27***) on the module are implemented.
